### ISSUE TITLE
Is the latest release 1.1.7 https://www.nuget.org/packages/AngleSharp.XPath/1.1.7 from NuGet not tagged in this GitHub repository?

### ISSUE THREAD

**Issue by martin-honnen** [AngleSharp/AngleSharp.XPath#26]
On NuGet, it seems the latest release of AngleSharp.XPath is 1.1.7, see https://www.nuget.org/packages/AngleSharp.XPath/1.1.7. However, when I look through this repository for tags https://github.com/AngleSharp/AngleSharp.XPath/tags, the latest I find is 1.1.5.

I use 1.1.7 in a .NET application and get a `System.NullReferenceException` on doing something like `angleSharpDoc.CreateNavigator(false).SelectSingleNode("//p").Prefix` or `angleSharpDoc.CreateNavigator(true).SelectSingleNode("//p").Prefix`.

Is there any place to see the sources for 1.1.7, to find out why that exception occurs?

**Comment by denis-ivanov** [comment]
Hello, @martin-honnen !
There were no changes except AngleSharp package update.
Can you show your HTML?

**Comment by martin-honnen** [comment]
The code was like
```
            var html = @"<html><head><title>Test</title></head><body><p id=p1>This is paragraph 1.<p>This is paragraph 2.</body></html>";

            var htmlDoc = new HtmlParser().ParseDocument(html);

            Console.WriteLine(htmlDoc.CreateNavigator().SelectSingleNode("//p").Prefix); 
```

when run against the 1.1.7 release it gives the error

System.NullReferenceException
  HResult=0x80004003
  Nachricht = Object reference not set to an instance of an object.
  Quelle = AngleSharp.XPath
  Stapelüberwachung:
   at AngleSharp.XPath.HtmlDocumentNavigator.get_Prefix()
   at PrefixBugAngleSharpXPath17.Program.Main(String[] args) in C:\Users\marti\source\repos\PrefixBugAngleSharpXPath17\Program.cs:line 15

But from your other comments on the other issues I understand that the problem is probably fixed on your development branch.

**Comment by denis-ivanov** [comment]
Fixed in `2.0.0-alpha-2` version.
